### PR TITLE
update FeedReader

### PR DESCRIPTION
**Pull request by jangernert**
- remove launchpad link
- update COPYING link to point to github
- update description (by now there are way more back-ends than ttrss and feedly)
- update icon
- update screenshot

### CODE REVIEW

File: apps/FeedReader.md [hunk 1]
```diff
@@ -19,24 +19,22 @@ links:
...
+    url: "https://github.com/jangernert/FeedReader/blob/master/COPYING"
 
 screenshots:
   - FeedReader.png
 
 installation:
   - system: elementary
     info:
-    - version: 0.3
+    - version: Stable
```
Comment (quassy) [inline apps/FeedReader.md]: `version` currently refers to the version number of the OS / elementary (so 0.3 was for Freya, 0.4 for Loki). Whether a PPA is "stable" or for "development" is determined by the `type`. Am I right that FeedReader currently support Freya _and_ Loki?

Comment (jangernert) [reply apps/FeedReader.md]: Oh, I see. The stable version of FeedReader supports both versions of the OS. But the development version (2.0) requires Gtk+ >= 3.20, thus supporting neither Freya nor Loki.

**Comment by quassy** [comment]
Thanks a lot for your pull request 😃! 

I will merge (and edit it a bit, see the code comment) it asap. Maybe you should also update your [description on Launchpad](https://launchpad.net/feedreader) to say that all development and bug reporting is now happening at GitHub.

**Comment by jangernert** [comment]
Updated!
Btw: is it useful to have a small list of features and some additional screenshots on the app-page? Or do you think it is enough if there is a link to the website with all the information?

**Comment by quassy** [comment]
I'll think about providing a small list of features / more detailed description of what each app is doing. A screenshot gallery, including short videos if available, is planned for the new version of elementary apps.

By the way, is the new screenshot up-to-date? My version of FeedReader (1.6.2) has a white / light grey sidebar not a black / dark grey one.

**Pull request merged**